unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/LoadingHUD/Bridge/LoadingBridge.cs

```diff
@@ -30,16 +30,16 @@ public void SetLoadingScreen(string jsonMessage)
 
         Payload payload = JsonUtility.FromJson<Payload>(jsonMessage);
 
-        if (payload.isVisible && !DataStore.i.HUDs.loadingHUD.fadeIn.Get() && !DataStore.i.HUDs.loadingHUD.visible.Get())
-            DataStore.i.HUDs.loadingHUD.fadeIn.Set(true);
+        if (payload.isVisible && !DataStore.i.loadingScreen.loadingHUD.fadeIn.Get() && !DataStore.i.loadingScreen.loadingHUD.visible.Get())
+            DataStore.i.loadingScreen.loadingHUD.fadeIn.Set(true);
 
-        if (!payload.isVisible && !DataStore.i.HUDs.loadingHUD.fadeOut.Get())
-            DataStore.i.HUDs.loadingHUD.fadeOut.Set(true);
+        if (!payload.isVisible && !DataStore.i.loadingScreen.loadingHUD.fadeOut.Get())
+            DataStore.i.loadingScreen.loadingHUD.fadeOut.Set(true);
 
         if (!string.IsNullOrEmpty(payload.message))
-            DataStore.i.HUDs.loadingHUD.message.Set(payload.message);
+            DataStore.i.loadingScreen.loadingHUD.message.Set(payload.message);
 
-        DataStore.i.HUDs.loadingHUD.showTips.Set(payload.showTips);
+        DataStore.i.loadingScreen.loadingHUD.showTips.Set(payload.showTips);
     }
 
     public void FadeInLoadingHUD(string jsonMessage)
```

Comment:
Let's mark old methods by `[Obsolete]`, it will be easier to detect them for removal in the future